rskj-core/src/main/java/co/rsk/config/RskSystemProperties.java

```diff
@@ -159,6 +161,29 @@ public double minerMinFeesNotifyInDollars() {
         return getDouble("miner.minFeesNotifyInDollars", 0);
     }
 
+    public boolean isRestServerEnabled() {
```

Comment:
Here we have 5 methods to return different properties related with the rest server and the HealthCheck module, maybe we can use one single objet go handle the properties.

Reply:
I've chosen to follow the way it's been done before, we can create an object too, let's discuss that.